beta/src/pages/learn/render-and-commit.md

```diff
@@ -65,28 +65,28 @@ export default function Image() {
 
 </Sandpack>
 
-Try commenting out the `ReactDOM.render` call and see the component disappear!
+`ReactDOM.render` कॉल पर कमेंट करने का प्रयास करें और देखें कि कौम्पोनॅन्ट गायब हो गया है|
 
-### Re-renders when state updates {/*re-renders-when-state-updates*/}
+### स्टेट अपडेट होने पर पुन: रेंडर {/*re-renders-when-state-updates*/}
 
-Once the component has been initially rendered, you can trigger further renders by updating its state with [`setState`](reference/setstate). Updating your component's state automatically queues a render. (You can imagine these as a restaurant guest ordering tea, dessert, and all sorts of things after putting in their first order, depending on the state of their thirst or hunger.)
+एक बार कौम्पोनॅन्ट शुरू में रेंडर किया गया है, तो आप [`setState`](reference/setstate) के साथ इसकी स्टेट को अपडेट करके आगे के रेंडरर्स को ट्रिगर कर सकते हैं|
```

Comment:
I think you missed this line to translate 

```
Updating your component's state automatically queues a render. (You can imagine these as a restaurant guest ordering tea, dessert, and all sorts of things after putting in their first order, depending on the state of their thirst or hunger.)
```

can you translate this also, seems this is part of line 72

Reply:
Queue will remain as it is right?

Reply:
I have translated the above line as below, any changes needed?
`अपने कौम्पोनॅन्ट की स्टेट को अपडेट करने पर ऑटोमेटिकली एक रेंडर केओए होता है। (आप उनकी प्यास या भूख की स्थिति के आधार पर, उनके पहले ऑर्डर में डालने के बाद चाय, मिठाई और सभी प्रकार की चीजों को ऑर्डर करने वाले एक रेस्तरां अतिथि के रूप में कल्पना कर सकते हैं।)`

Reply:
> केओए होता है।

`केओए होता है।` => `रेंडर कियु हो जाता है।`